doc/topics/windows/multi-minion.rst

```diff
@@ -0,0 +1,164 @@
+placed the root_dir in a location that the user does not have access to, you'll
+need to give the user full permissions to that directory.
+
+.. code-block:: powershell
+
+    $RootDir = "<new root_dir location>"
+    $User    = "<user running salt>"
+    $acl = Get-Acl -Path "$RootDir"
+    $access_rule = New-Object System.Security.AccessControl.FileSystemAccessRule($User, "Modify", "Allow")
+    $acl.AddAccessRule($access_rule)
+    Set-Acl -Path "$RootDir" -AclObject $acl
+
+3. Create directory structure
+-----------------------------
+
+Salt expects a certain directory structure to be present to avoid unnecessary
+messages in the logs. This is usually handled by the installer. Since we're
+running our own instance, we need to do it. Make sure the following directories
+are present:
+
+  - root_dir\\conf\\minion.d
+  - root_dir\\conf\\pki
+  - root_dir\\var\\log\\salt
+  - root_dir\\var\\run
+  - root_dir\\var\\cache\\salt\\minion\\extmods\\grains
+  - root_dir\\var\\cache\\salt\\minion\\proc
+
+.. code-block:: powershell
+
+    $RootDir = "<new root_dir location>"
+    $cache_dir = "$RootDir\var\cache\salt\minion"
+    New-Item -Path "$RootDir\conf" -Type Directory
+    New-Item -Path "$RootDir\conf\minion.d" -Type Directory
+    New-Item -Path "$RootDir\conf\pki" -Type Directory
+    New-Item -Path "$RootDir\var\log\salt" -Type Directory
+    New-Item -Path "$RootDir\var\run" -Type Directory
+    New-Item -Path "$cache_dir\extmods\grains" -Type Directory
+    New-Item -Path "$cache_dir\proc" -Type Directory
+
+4. Write minion config
+----------------------
+
+The minion will need is own config, separate from the system minion config. This
+config tells the minion where everything is as well as defines the master and
+minion id. Create a minion config file named ``minion`` in the conf directory.
+
+.. code-block:: powershell
+
+    New-Item -Path "$env:LocalAppData\Salt Project\Salt\conf\minion" -Type File
+
+Make sure the config file has at least the following contents:
+
+.. code-block:: yaml
+
+    master: <ip address, dns name, etc>
+    id: <minion id>
+
+    root_dir: <root_dir>
+    log_file: <root_dir>\val\log\salt\minion
+    utils_dirs:
+      - <root_dir>\var\cache\salt\minion\extmods
+    winrepo_dir: <root_dir>\srv\salt\win\repo
+    winrepo_dir_ng: <root_dir>\srv\salt\win\repo-ng
+
+    file_roots:
+      base:
+        - <root_dir>\srv\salt
+        - <root_dir>\srv\spm\salt
+
+    pillar_roots:
+      base:
+        - <root_dir>\srv\pillar
+        - <root_dir>\srv\spm\pillar
+
+    thorium_roots:
+      base:
+        - <root_dir>\srv\thorium
+
+Run the minion
+--------------
+
+Everything is now set up to run the minion. You can start the minion as you
+would normally, but you need to specify the full path to the config file you
+created above.
+
+.. code-block:: powershell
+
+    salt-minion.exe -c <root_dir>\conf
+
+Register the minion as a service
```

Comment:
**Comment (consistency):** Same comment. Do you want to continue numbering here?